extensions/transform/org.eclipse.smarthome.transform.xpath/README.md

```diff
@@ -77,18 +77,18 @@ rule "Convert XML to Item Type Number"
     Item Temperature_xml changed
  then
     // use the transformation service to retrieve the value
-	// Simple
-	val mytest = transform("XPATH", "/*[name()='PTZStatus']
-									 /*[name()='AbsoluteHigh']
-									 /*[name()='azimuth']
-									 /text()", 
-									 Temperature_xml.state.toString )  
-	// Fully qualified
-	val mytest = transform("XPATH", "/*[local-name()='PTZStatus'    and namespace-uri()='http://www.hikvision.com/ver20/XMLSchema']
-									 /*[local-name()='AbsoluteHigh' and namespace-uri()='http://www.hikvision.com/ver20/XMLSchema']
-									 /*[local-name()='azimuth'      and namespace-uri()='http://www.hikvision.com/ver20/XMLSchema']
-									 /text()",
-									 Temperature_xml.state.toString )
+    // Simple
+    val mytest = transform("XPATH", "/*[name()='PTZStatus']
+				     /*[name()='AbsoluteHigh']
```

Comment:
In order to achieve really robust rendering it would make sense replaceing _all_ tabs within this code block with spaces, wouldn't it?